docs/units/interoperable-data/topics/3-vocabularies-and-ontologies.md

```diff
@@ -99,58 +90,22 @@ terms4FAIRskills:
- 
-
-## Topic, definition and scope
-
-
-
-* What are ontologies and when/why do we need them
-* What selection criteria do we use to chose ontologies and vocabularies
-* What types of vocabularies there are
-* Create data vocabularies from scratch / apply existing ones
-* How do use ontologies in real life (practical demonstration)
-
-
----
-
-## FAIR element(s)
-
-* Findability: An ontology can improve findability by providing a standardized vocabulary for describing data. By using established terms and relationships, researchers can ensure that their data is discoverable by others who are searching for specific concepts or entities.
-
-* Accessibility: An ontology can enhance accessibility by providing a clear and well-defined structure for organizing and representing data. By adhering to ontology standards, researchers can make their data more accessible to both humans and machines, enabling easier understanding and interpretation.
+## Overview
 
-* Interoperability: An ontology plays a crucial role in achieving interoperability by providing a shared understanding of concepts and their relationships. By using standardized terms and relationships from an ontology, researchers can enable data integration and exchange across different systems and domains.
+### Topic
 
-* Reusability: An ontology supports reusability by providing a common vocabulary that can be used across multiple datasets and applications. By using established ontologies, researchers can ensure that their data is reusable and can be easily integrated with other datasets for further analysis or comparison.
-
----
-
-## Summary of Tasks / Actions
-
-* Browse the [BioPortal](https://bioportal.bioontology.org/) and [Linked Open Vocabularies (LOV)](https://lov.linkeddata.es/dataset/lov) for general overview and understanding of ontologies. 
-
-* e.g. Search and filter the [>800 terminologies](https://fairsharing.org/search?fairsharingRegistry=Standard&recordType=terminology_artefact&page=1) listed within FAIRsharing accordning to a selected criteria.
-* …
+This lesson plan will help you better understand what ontologies and vocabularies are, how they relate to the FAIR principles, and why they are important for research. At the end of this lesson, participants will be able to define and distinguish ontologies and vocabularies, understand how they are used in repositories, explain their role in metadata organisation, use ontology lookup services, and evaluate the process of creating an ontology or vocabulary.
```

Comment:
The first sentence is learner-oriented, while the rest is not. I would try not to mix the style of these sentences.